opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -2103,8 +2103,11 @@ const adaptFilterToComputedReliabilityFilterKey = async (context, user, filter)
 const adaptFilterToWorkflowFilterKey = async (context, user, filter) => {
   const { key, mode = 'or', operator = 'eq', values } = filter;
   const arrayKeys = Array.isArray(key) ? key : [key];
-  if (arrayKeys[0] !== WORKFLOW_FILTER || arrayKeys[0] !== X_OPENCTI_WORKFLOW_ID || arrayKeys.length > 1) {
-    throw UnsupportedError(`A filter with these multiple keys is not supported: ${arrayKeys}`);
+  if (arrayKeys.length > 1) {
+    throw UnsupportedError(`A filter with these multiple keys is not supported : ${arrayKeys}`);
+  }
+  if (!arrayKeys[0].includes(WORKFLOW_FILTER, X_OPENCTI_WORKFLOW_ID)) {
```

Comment:
should be `![WORKFLOW_FILTER, X_OPENCTI_WORKFLOW_ID].includes(arrayKeys[0])`
String.includes() can work only for one parameter